Rubberduck.Core/UI/Converters/ImageSourceConverter.cs

```diff
@@ -12,17 +12,19 @@ public abstract class ImageSourceConverter : IValueConverter
     {
         protected static ImageSource ToImageSource(Image source)
         {
-            var ms = new MemoryStream();
+            using (var ms = new MemoryStream())
+            {
+                ((Bitmap) source).Save(ms, System.Drawing.Imaging.ImageFormat.Png);
+                var image = new BitmapImage();
+                image.BeginInit();
+                image.CacheOption = BitmapCacheOption.OnLoad;
```

Comment:
This probably explains the increased memory consumption; the bitmaps weight more than the leaked object pointers ;-)